### DISCUSSION TITLE
external yaml files in header

### DISCUSSION THREAD

**Discussion by wahalulu** [quarto-dev/quarto-cli#2501] [Q&A] [answered]
I have a whole bunch of qmd files that share the same output format (reveal.js slides) with a high degree of customization. Is there a way to store the `format:` part of the header in a separate YAML file (or any other shared YAML block in the header) and include it in the header for each individual qmd? Is there a more efficient way of doing this?

**Comment by dragonstyle** [comment, marked as answer]
I think either a quarto project or `_metadata.yml` files might suit your needs. More here:

https://quarto.org/docs/projects/quarto-projects.html#shared-metadata

Let us know if this gets you going!
